### ISSUE TITLE
sites opened in standalone missing accass to flash etc 

### ISSUE THREAD

**Issue by thomasvjohansen** [autonome/standalone#8]
great addon, perfect use for cloud services, to get em looking like apps.

though tried it with a music cloud service which uses flash.
Even though firefox can handle it, the standalone addon cant.

see screenshot
![firefox-standalone-addon-noaccess to flash](https://cloud.githubusercontent.com/assets/15713353/11019688/43821e82-8604-11e5-8f77-722f534ef4d1.png)

**Comment by amjames** [comment]
This is defiantly a big issue, I primarily use this kind of service to run the spotify web player as a stand-alone app, but with add-block enabled. If there is no access to the flash plugin I imagine that there is no access to add-block as well.  Hes the developer dropped this project? The last commit was 3 months ago :(

**Comment by autonome** [comment]
Yeah, just haven't had time to work on it... need more developers ;)
